policyengine_uk/variables/gov/hmrc/tax_free_childcare/is_eligible_child_for_tax_free_childcare.py

```diff
@@ -0,0 +1,24 @@
+from policyengine_uk.model_api import *
+
+
+class is_eligible_child_for_tax_free_childcare(Variable):
```

Comment:
Can we make this `is_child_receiving_tax_free_childcare` since it's receipt rather than eligibility? Same for the others